Sources/Parameters.swift

```diff
@@ -18,11 +18,11 @@ final class ValueParameter {
 
 final class SideParameter {
 
-    unowned let view: UIView
```

Comment:
Why did you remove `unowned`? Same for other places like this in `Relations.swift`.

Reply:
because this `view` replaced by view type that is not class